theories/_CoqProject

```diff
@@ -1,7 +1,7 @@
 -docroot .
 
 -arg -w -arg -notation-overridden
-COQDOCFLAGS = "--charset utf-8 -s --with-header ../website/resources/header.html --with-footer ../website/resources/footer.html --index indexpage"
+COQDOCFLAGS = "--charset utf-8 -s --with-header ../website/resources/header.html --with-footer ../website/resources/footer.html --index indexpage --external https://hermesmarc.github.io/coq-library-fol/website/undec Undecidability"
```

Comment:
@dominik-kirst this link is not supposed to be there, right?

Reply:
changed to the uds-psl websites now, integrates the undec html with the fol html